taxi/views.py

```diff
@@ -1,3 +1,13 @@
+from django.http import HttpResponse
 from django.shortcuts import render
+from .models import Car, Driver, Manufacturer
```

Comment:
Add blank line between different import groups
Use absolute import instead of relative